rst_files/julia_by_example.rst

```diff
@@ -11,16 +11,14 @@ Introductory Examples
 Overview
 ==================
 
-
-We're now ready to start learning the Julia language itself
-
+We're now ready to start learning the Julia language itself 
 
 Level
 -------
 
 Our approach is aimed at those who already have at least some knowledge of programming --- perhaps experience with Python, MATLAB, Fortran, C or similar
 
-In particular, we assume you have some familiarity with fundamental programming concepts such as
+In particular, we assume you have some familiarity with fundamental programming concepts such as:
```

Comment:
We can't add on colons, it is not the style of the lectures.

Reply:
```suggestion
In particular, we assume you have some familiarity with fundamental programming concepts such as
```